src/toasts/AnalyticsToast.tsx

```diff
@@ -23,16 +23,52 @@ import Analytics from "../Analytics";
 import AccessibleButton from "../components/views/elements/AccessibleButton";
 import GenericToast from "../components/views/toasts/GenericToast";
 import ToastStore from "../stores/ToastStore";
+import {
+    ButtonClicked,
+    showDialog as showAnalyticsLearnMoreDialog,
+} from "../components/views/dialogs/AnalyticsLearnMoreDialog";
+import { Action } from "../dispatcher/actions";
 
 const onAccept = () => {
     dis.dispatch({
-        action: 'accept_cookies',
+        action: Action.PseudonymousAnalyticsAccept,
     });
 };
 
 const onReject = () => {
     dis.dispatch({
-        action: "reject_cookies",
+        action: Action.PseudonymousAnalyticsReject,
+    });
+};
+
+const onLearnMoreNoOptIn = () => {
+    showAnalyticsLearnMoreDialog({
+        onFinished: (buttonClicked?: ButtonClicked) => {
+            if (buttonClicked === ButtonClicked.Primary) {
+                // user clicked "Enable"
+                onAccept();
+            }
+            // otherwise, the user either clicked "Cancel", or closed the dialog without making a choice,
+            // leave the toast open
+        },
+        primaryButton: _t("Enable"),
+    });
+};
+
+const onLearnMorePreviouslyOptedIn = () => {
+    showAnalyticsLearnMoreDialog({
+        onFinished: (buttonClicked?: ButtonClicked) => {
+            if (buttonClicked === ButtonClicked.Primary) {
+                // user clicked "That's fine"
+                onAccept();
```

Comment:
I might be blind, but where does `onAccept` come from?

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/6936/files/560494ff65cbc63edd590496c57da850e20099e6#diff-4386d47cc2349bae17a0f01e44b162d774569285c3987d6374334fd9f0bff3ccR32

Reply:
ah ha, thanks!